install/class.Setup.php

```diff
@@ -154,6 +154,10 @@ public function __invoke($params)
         }
 
         $global = $parameters['configuration']['global'];
+
+        $markers = self::getConfigurationMarkers();
```

Comment:
nitpick: `getConfigurationMarkers` is not a static method, so you should use `$this` here

```suggestion
        $markers = $this->getConfigurationMarkers();
```

Reply:
Good catch Hector, but probably this method should be static so better to change the declaration to static.

Reply:
The signature for [__invoke](https://www.php.net/manual/en/language.oop5.magic.php#object.invoke) is dictated by PHP itself and it should never be static. It is triggered automatically by PHP when the object is used as a function and we are surely relying on that on the install scripts. In PHP 8 this will produce [a fatal error](https://php.watch/versions/8.0/non-static-static-call-fatal-error).

Regarding `getConfigurationMarkers()`, I'd prefer keeping it as non-static as the caller is also non-static.

Reply:
Changed, thx for catch